### DISCUSSION TITLE
I need help on the Skiovox exploit and I am trying to do Stage 2A

### DISCUSSION THREAD

**Discussion by Vhirttuegaming** [3kh0/ext-remover#881] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
I currently have a back button on my network error screen. 
Does 2A require you to sign out instead of restart?
If it requires you to sign out then I have a problem I don't see apps on the bottom left corner
If it requires you to restart then It was fine until I reached 2A 
which then when I tried to press escape and enter it requires me to use my email and after 2 or 3 seconds it disappears and leads me back to the error screen. 
Any help?

**Comment by AshtonDavies** [comment]
I'm confused because I don't remember any sign out or restart step in 2A but I can double check the official instructions.

Edit: Yeah, there's no signing out in 2A. You're supposed to add other Wi-Fi network, press Esc key twice to take you to the password screen with your password still entered, then press enter, all done within 4 secs of clicking "Add other Wi-Fi network." You can ask me if you need any more help with SKIOVOX.

**Reply by Vhirttuegaming** [reply]
does it need me I sign out or restart? If restart my start up screen. If sign out my account screen appears. which one do I do first btw signing out of my account doesn't give me "apps". So im confused on whether or not you need to sign out or restart because they have very specific purposes. I tried to restart then everything was fine until when I got to the escape and enter part it requires me to actually type my actual email which after 2-3 seconds it disappears and I cannot do it anymore.
Skiovox requires you to do one of those two options and in the instruction they are not very specific on which one to actually do. 
Page 8. "First, sign out or restart your Chromebook to get
to the login screen."
Edit: I realized that both sign out and restart do work the same but this actually logs out of my account which makes the escape and enter part hard

**Reply by AshtonDavies** [reply]
How are you supposed to launch a kiosk application signed in?

**Reply by Vhirttuegaming** [reply]
Please list the steps for 2A I’m trying my best to read the instructions I feel like the instructions are kind of contradicting itself or that it doesn’t give me specific details.

**Reply by Vhirttuegaming** [reply]
So what I know is that if you sign or restart it brings you to the start up screen everything is ok until I’m at page 16 of the instructions. Where when I do the steps it wants me sign up to my email and after like 2-3 seconds it goes away. What really makes me confused is that page 15 
Tells me to already typed in my password. When I sign out or restart to go to the start up screen I typed in my email and then the apps disappears until I go back.
Legend: A: logged in before doing it B: not logged in
The steps I did are as follows.
If logged in:
A1: Sign out or restart 
Or Else or after A1: 
A2 or B1: turn off WiFi
A3 or B2: Press app button
A4 or B3: Press one of the apps
*Instantly*
A5 or B4: Alt+Shift+s
*Wait*
A6 or B5: Accessibility button
A7 or B6: Question mark
A8 or B7: Add other WiFi network
A9 or B8: Press Escape key twice
A10 or B9: Press Enter
*Please sign in using email*
2-3 seconds
*Disappears*

**Reply by AshtonDavies** [reply]
What version are you on? Maybe it's not even the version. It stopped working for me the day after I did the exploit on someone's Chromebook out of nowhere and I don't think it got signed in and updated between that. Are the official instructions confusing for you?

**Reply by Vhirttuegaming** [reply]
The instructions are okay buts just that it is not working for me and that the instructions just says random stuff like “You should already have your email typed in” even though I’m clearly signed out or logged out. What they say I should do and my computer’s incapability to do that is a cause. I keep having a feeling that I’m dumb cus I cannot do it cus I didn’t follow the instructions, but I did follow the instructions.

**Reply by Vhirttuegaming** [reply]
It is 1.15

**Reply by AshtonDavies** [reply]
If you have a password on the sign in screen (usernames & photos enabled), have it typed and don't press enter. Later on, you should be taken back to that screen or something I don't remember exactly.

Just in case your legend is that you're signing out in the middle.

**Reply by Vhirttuegaming** [reply]
The sign in screen when I did escape and enter or the sign in screen or when I restarted? Can you show me the mistake I made? It is also helpful if you can list the steps.

**Reply by AshtonDavies** [reply]
I mean, the steps are already listed on skiovox.com which is what you're reading from. I'm not sure what you mean by the two different sign-in screens. If this helps, try my cleaner instructions (https://chrose.netlify.app/detail/SKIOVOX).

**Reply by AshtonDavies** [reply]
I also want to give a note to anyone trying to perform this exploit that admins can prevent the use of SKIOVOX. ![image.png](https://github.com/3kh0/ext-remover/assets/117958718/675461d0-ff16-41c5-95be-846306451bca)

**Reply by HeroLink6** [reply]
If your Chromebook is in version 119 or higher, then it has been patched. (At least for Stage B)

**Reply by Vhirttuegaming** [reply]
the sign in screen when I restart from A1 and the sign in screen for my email of before A10 or B9.

**Reply by Vhirttuegaming** [reply]
do you have to do the exploit every time when you want to go to private browsing? is there no shortcut?

**Reply by AshtonDavies** [reply]
> do you have to do the exploit every time when you want to go to private browsing? is there no shortcut?

No, there is no shortcut to perform this, but you can leave it running with your lid closed if you can somehow still do your school work in the SKIOVOX environment.

**Reply by Vhirttuegaming** [reply]
are you looking forward into the issue?

**Reply by Vhirttuegaming** [reply]
guys when I try to do escape two times and enter it doesn't work.

**Reply by Vhirttuegaming** [reply]
so the question is how can you directly go to your account after you signed out?

**Reply by AshtonDavies** [reply]
> so the question is how can you directly go to your account after you signed out?

You have to perform SKIOVOX again, but your profile data should be saved.

**Reply by Vhirttuegaming** [reply]
No No No, I have a new chromebook that has the back button so I have to do 2A and when I try to do the esc and enter part after clicking on the add network but its not working if it works then it sends me to add email which vanishes after 1 second.

**Comment by S-PScripts** [comment]
Closed unless you still the ability to do Skiovox

**Comment by schoolexploitkid** [comment]
Skiovox is patched as of v132

**Reply by akabutnicer** [reply]
skiovox in the big 2025 is crazy

**Reply by akabutnicer** [reply]
leave that in 2023 (early 2024 tops)

**Reply by SlitheringLizard1** [reply]
lol I did skiovox 2 weeks ago
